jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/BaseImageParameters.java

```diff
@@ -53,6 +56,16 @@ public BaseImageParameters(ObjectFactory objectFactory) {
   @Nested
   @Optional
   public ListProperty<PlatformParameters> getPlatforms() {
+    String property = System.getProperty(PropertyNames.FROM_PLATFORMS);
+    if (property != null) {
+      List<PlatformParameters> parsed =
+          ConfigurationPropertyValidator.parseListProperty(property).stream()
+              .map(PlatformParameters::of)
+              .collect(Collectors.toList());
+      if (!parsed.equals(platforms.get())) {
```

Comment:
`.equals()` wasn't working because `PlatformParameters` didn't define `.equals()` and `.hashCode()`. I fixed it.

Reply:
Thank you, missed it.